app/components/agenda/agendaitem/agendaitem-case-panel/agendaitem-case-panel-edit.hbs

```diff
@@ -105,14 +105,25 @@
           <AuIcon @icon="lock-closed" />
           {{t "private-comment-title"}}
         </AuLabel>
-        <AuTextarea
-          data-test-agendaitem-titles-edit-private-comment={{true}}
-          id="private-comment-title"
-          rows="7"
-          value={{@agendaitem.privateComment}}
-          @width="block"
-          {{on "input" (pick "target.value" (set @agendaitem "privateComment"))}}
-        />
+        {{#if this.internalReview}}
```

Comment:
My original thought was to check the CABINET_SUBMISSIONS feature flag here as well. 
With the current way 2 systems can potentially be used at the same time...

Of course then the internalReview would need to be created if it doesn't exist... 

This might be a non-issue, as with your case we'll capture all `agendaitems` that were created as `subcases` or `submission` from the moment we deploy this, which is fine.

Reply:
I will run a test to see what this does with the submissions flag off.
Since I tested adding this model to subcases that have no submissions I expect it to work just fine.